src/main/java/ezschedule/ui/EventListPanel.java

```diff
@@ -31,7 +31,7 @@ public EventListPanel(ObservableList<Event> eventList) {
     }
 
     /**
-     * Custom {@code ListCell} that displays the graphics of a {@code Person} using a {@code PersonCard}.
+     * Custom {@code ListCell} that displays the graphics of a {@code Event} using a {@code PersonCard}.
```

Comment:
`{@code PersonCard}` should be changed too!

Reply:
Fixed